packages/vite/src/rsc/rscStudioHandlers.ts

```diff
@@ -0,0 +1,171 @@
+import http from 'node:http'
+import type { PassThrough } from 'node:stream'
+
+import type { Request } from 'express'
+
+import { getRawConfig, getConfig } from '@redwoodjs/project-config'
+
+import { renderRsc } from './rscWorkerCommunication.js'
+import type { RenderInput } from './rscWorkerCommunication.js'
+
+const isTest = () => {
+  return process.env.NODE_ENV === 'test'
+}
+
+// TODO (RSC): This should be !== 'production'
+// but since RSC apps currently run in production mode
+// we need to check for 'production' instead of 'development'
+// for now when sending to Studio
+const isDevelopment = () => {
+  // return process.env.NODE_ENV !== 'production' && !isTest()
+  return process.env.NODE_ENV === 'production' && !isTest()
```

Comment:
Can we make this `!==` please? So that the function does what its name says? 
And then have different logic in `shouldSendToStudio` instead

Reply:
That makes more sense to do. Will do.